unit_tests/test_scylla_yaml.py

```diff
@@ -398,7 +398,8 @@ def test_scylla_yaml(self):
                 'virtual_dirty_soft_limit': None,
                 'volatile_system_keyspace_for_testing': None,
                 'workdir': None,
-                'write_request_timeout_in_ms': None
+                'write_request_timeout_in_ms': None,
+                'enable_tablets': None
```

Comment:
Care to add comma to avoid redundant changes like you needed with line 401?

Reply:
done